Src/Support/GoogleApis.Core/Apis/Logging/BaseLogger.cs

```diff
@@ -0,0 +1,182 @@
+﻿/*
+Copyright 2011 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Util;
+using System;
+using System.Globalization;
+
+namespace Google.Apis.Logging
+{
+    /// <summary>
+    /// An abstract base logger, upon which real loggers may be built.
+    /// </summary>
+    public abstract class BaseLogger : ILogger
+    {
+        // Matches gRPC datetime log format
+        private const string DateTimeFormatString = "MMdd HH:mm:ss.ffffff";
```

Comment:
OK, I agree yyyy-MM-dd is better.
It's fairly possible that gRPC logs and logs from here will end up in the same log file, so being consistent in formatting will make parsing simpler. But I guess it's fairly simple to differentiate between these two formats.
Changed.